docs/deploying.md

```diff
@@ -69,119 +69,119 @@ Each service's `main` package is located in the `cmd` directory.
+   (Cloud Build GitHub App). You must choose a repository on which you have
+   administrator permissions.
 
-1.  Execute Terraform:
+   1. Make a note of the repository you have chosen. You must set the
+   repository name (for example, 'exposure-notification-server') and owner
+   (for example 'google') as variables when running the `terraform apply`
+   command.
 
-    Without Cloud Build Triggers:
+1. Authenticate with Google Cloud using the `gcloud` command-line tool:
 
-    ```text
-    $ terraform apply \
-        -var project=$PROJECT_ID
-    ```
+   ```console
+   gcloud auth login && gcloud auth application-default login
+   ```
 
-    With Cloud Build Triggers:
+   This will open two authentication windows in your web browser.
 
-    ```text
-    $ terraform apply \
-        -var project=${PROJECT_ID} \
-        -var region="us-central-1" \
-        -var use_build_triggers=true \
-        -var repo_owner=${YOUR_REPO_OWNER} \
-        -var repo_name=${YOUR_REPO_NAME}
-    ```
+   You may need to `unset GOOGLE_APPLICATION_CREDENTIALS` as it takes precedence
+   over gcloud's login settings.
 
-Terraform will begin by creating the service accounts and enabling the services
-on GCP which are required to run this server.
+1. Change to this directory and run `terraform init`.  Terraform will
+   automatically download the plugins required to execute this code.
 
-> NOTE: This configuration assumes production scale. The scale of this means
-> a substantial billed amount. You can downsize this to save on costs
-> For example you can set other vars in terraform apply to smaller values.
-> ```
->  $ terraform apply \
->       -var project=${PROJECT_ID} \
->       -var region="us-central-1" \
->       -var use_build_triggers=true \
->       -var repo_owner=${YOUR_REPO_OWNER} \
->       -var repo_name=${YOUR_REPO_NAME} \
->       -var cloudsql_tier="db-custom-1-3840" \
->       -var cloudsql_disk_size="16"
-> ```
+1. Run `terraform apply` to start deployment:
 
+   Without Cloud Build Triggers:
 
-electin vCPU and Postgres size (concurrent connections):
+   ```console
+   terraform apply \
+     -var project=$PROJECT_ID
+   ```
 
-PostgresSql sizing
-And pricing
-https://cloud.google.com/sql/docs/postgres/create-instance
-Combined with connection limits: https://cloud.google.com/sql/docs/quotas#cloud-sql-for-postgresql-connection-limits
-Choice for now:
+   With Cloud Build Triggers:
 
-db-custom-8-30720
+   ```console
+   terraform apply \
+     -var project=${PROJECT_ID} \
+     -var region="us-central-1" \
+     -var use_build_triggers=true \
+     -var repo_owner=${YOUR_REPO_OWNER} \
+     -var repo_name=${YOUR_REPO_NAME}
+   ```
 
+Terraform will begin by creating the service accounts and enabling the services
+on Google Cloud that are required to run this server.
 
-30 gb which gives 500 concurrent connections
+#### High traffic example deployment
 
+**Important**: This is a production-ready, high traffic sample deployment.
```

Comment:
```suggestion
**Important**: The default configuration will create a production-ready, high traffic deployment.
```

Reply:
Believe both of these edits have been incorporated in the latest branch commit. PTAL, cheers